tests/test_device.py

```diff
@@ -167,7 +167,7 @@ def get_device(wires=1):
 
 
 @pytest.fixture(scope="function")
-def mock_device_arbitrary_wires(monkeypatch, wires):
+def mock_device_arbitrary_wires(monkeypatch):
```

Comment:
Should we rename this with `fixture` and add a name to the decorator as with the other fixtures?

Reply:
I'm not sure I follow, sorry. `lint` does not complain about the name being redefined, which it does for fixtures that do not have `scope="function"`.

Reply:
Ahh ok! Sorry I misunderstood your initial comment. If its not complaining then we should be fine 👍🏼